packs/feats/halfling-luck.json

```
@@ -28,7 +28,33 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "outcome": [
+                    "failure",
+                    "criticalFailure"
+                ],
+                "predicate": [
+                    {
+                        "gt": [
+                            "halfling-luck",
+                            0
+                        ]
+                    }
+                ],
+                "selector": [
+                    "skill-check",
+                    "saving-throw"
+                ],
+                "text": "You can re-roll this check or saving throw.",
+                "title": "Halfling Luck"
```

```suggestion
                "text": "{item|description}",
                "title": "<span class=\"action-glyph\">f</span> {item|name}"
```
Text in a rule element should either just be a reference back to the item description and name, or localized.